Moli/Core/Screens/Profile/ProfileImageAndBannerView.swift

```diff
@@ -0,0 +1,101 @@
+import SwiftUI
+import Mantis
+
+enum ImageType {
+    case profile, banner
+}
+
+struct ProfileImageAndBannerView: View {
+    @EnvironmentObject private var myUser: MyUserDataModel
+    @StateObject var profileImagesDataModel = ProfileImagesDataModel()
+
+    var bannerImages: API.User.BannerThumbnailContainer?
+    var profileUserImage: API.User.UserMinified
+    var isMe: Bool
+    private let profileImageSizeYOffset: CGFloat = .userImageSizeMax / 2
+
+    private func onImageCropped (_ image: UIImage) {
+        Task {
+            await profileImagesDataModel.onImageCroppedSelected(image)
+            await myUser.user.refreshProfile()
+        }
+    }
+
+    private func onRemoveImage () {
+        Task {
+            await profileImagesDataModel.removeImage()
+            await myUser.user.refreshProfile()
+        }
+    }
+
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            HStack(spacing: 0) {
+                BannerImageView(bannerImages: bannerImages, isMe: isMe)
+            }
+            .frame(height: .bannerImageHeight)
+            .frame(maxWidth: .infinity)
+            .background(Color.gray50)
+            .onTapGesture {
+                if isMe {
+                    profileImagesDataModel.onImageTap(.banner)
+                }
+            }
+
+            VStack(spacing: 0) {
+                UserThumbnailView(userMinified: profileUserImage, size: .max)
+            }
+            .overlay(Circle().strokeBorder(Color.primaryDark, lineWidth: 3, antialiased: true))
+            .offset(x: 0, y: profileImageSizeYOffset)
+            .onTapGesture {
+                if isMe {
+                    profileImagesDataModel.onImageTap(.profile)
+                }
+            }
+        }
+        .padding(.bottom, profileImageSizeYOffset)
+        .confirmationDialog(
+            "Are you sure?",
+            isPresented: $profileImagesDataModel.shouldShowPhotoConfirmDialog
+        ) {
+            Button("Choose image from library") {profileImagesDataModel.chooseImageFromLibrary()}
+            .padding()
+
+            Button("Take a photo") { profileImagesDataModel.takeCameraImage() }
+            .padding()
+
+            Button("Remove current image", action: onRemoveImage).padding()
```

Comment:
:art: Här kan du ha .destructive role:n för att göra knappen röd-textad
```suggestion
            Button("Remove current image", role: .destructive, action: onRemoveImage)
```